driver/src/main/java/org/kaazing/k3po/driver/internal/netty/channel/ChannelAddress.java

```diff
@@ -50,6 +52,7 @@ public int compare(ChannelAddress o1, ChannelAddress o2) {
     private final boolean ephemeral;
 
     private final ChannelAddress transport;
+    private final Map<String, Object> options;
```

Comment:
Recommend keeping generic options out of the general purpose ChannelAddress API, and instead add specific options strongly typed to FileChannelAddress since they are only consumed by the file transport.